terraform/modules/apps/publishing-api/main.tf

```diff
@@ -7,82 +7,161 @@ terraform {
-  }
+data "aws_secretsmanager_secret" "sentry_dsn" {
+  name = "publishing_api_app-SENTRY_DSN"
 }
 
-#
-# Dependencies
-#
 
-data "aws_security_group" "service_dependencies" {
-  id = "sg-05ad7398fc0d7c5b4" # legacy govuk-aws group: govuk_publishing-api_access
+module "app" {
+  source                           = "../../app"
+  cpu                              = "512"
+  memory                           = "1024"
+  vpc_id                           = var.vpc_id
+  cluster_id                       = var.cluster_id
+  service_name                     = var.service_name
+  subnets                          = var.private_subnets
+  mesh_name                        = var.mesh_name
+  service_discovery_namespace_id   = var.service_discovery_namespace_id
+  service_discovery_namespace_name = var.service_discovery_namespace_name
+  task_role_arn                    = var.task_role_arn
+  execution_role_arn               = var.execution_role_arn
+  extra_security_groups            = [var.govuk_management_access_security_group]
+  container_definitions = [
+    {
+      "name" : "publishing-api",
+      "image" : "govuk/publishing-api:with-content-schemas",
+      "essential" : true,
+      "environment" : [
+        # TODO: factor our hardcoded stuff
+        { "name" : "APPMESH_VIRTUAL_NODE_NAME", "value" : "mesh/${var.mesh_name}/virtualNode/${var.service_name}" },
+        { "name" : "CONTENT_API_PROTOTYPE", "value" : "yes" },
+        { "name" : "CONTENT_STORE", "value" : "http://content-store.${var.service_discovery_namespace_name}" },
+        { "name" : "DRAFT_CONTENT_STORE", "value" : "https://draft-content-store.${var.service_discovery_namespace_name}" },
+        { "name" : "EVENT_LOG_AWS_ACCESS_ID", "value" : "AKIAJE6VSW25CYBUMQJA" },
+        { "name" : "EVENT_LOG_AWS_BUCKETNAME", "value" : "govuk-publishing-api-event-log-test" },
+        { "name" : "EVENT_LOG_AWS_USERNAME", "value" : "govuk-publishing-api-event-log_user" },
+        { "name" : "DEFAULT_TTL", "value" : "1800" },
+        { "name" : "GOVUK_APP_DOMAIN", "value" : var.service_discovery_namespace_name },
+        { "name" : "GOVUK_APP_DOMAIN_EXTERNAL", "value" : var.govuk_app_domain_external },
+        { "name" : "GOVUK_APP_NAME", "value" : var.service_name },
+        { "name" : "GOVUK_APP_TYPE", "value" : "rack" },
+        { "name" : "GOVUK_CONTENT_SCHEMAS_PATH", "value" : "/govuk-content-schemas" },
+        { "name" : "GOVUK_GROUP", "value" : "deploy" }, # TODO: clean up?
+        { "name" : "GOVUK_STATSD_PREFIX", "value" : "fargate" },
+        { "name" : "GOVUK_USER", "value" : "deploy" }, # TODO: clean up?
+        { "name" : "GOVUK_WEBSITE_ROOT", "value" : var.govuk_website_root },
+        { "name" : "PLEK_SERVICE_CONTENT_STORE_URI", "value" : "http://content-store.${var.service_discovery_namespace_name}" },
+        { "name" : "PORT", "value" : "80" },
+        { "name" : "RABBITMQ_HOSTS", "value" : "rabbitmq.test.${var.service_discovery_namespace_name}" },
```

Comment:
This doesn't look right to me - the MQ isn't on the service mesh domain.

Reply:
I'm going to use an internal domain, we can move it to a service mesh once rabbitmq is on the mesh.

Reply:
Yeah I think it should be `rabbitmq.test.govuk-internal.digital` for the test environment initially.

We won't ever be running RabbitMQ on ECS - we'll be moving to either Amazon MQ or SQS.

Reply:
cool.